src/macaron/config/defaults.py

```diff
@@ -20,15 +20,23 @@ def get_list(
         self,
         section: str,
         item: str,
-        delimiter: Optional[str] = None,
-        fallback: Optional[list] = None,
+        delimiter: Optional[str] = "\n",
+        fallback: Optional[list[str]] = None,
         duplicated_ok: bool = False,
-    ) -> list:
-        """Parse and return a list of strings from an item in ``defaults.ini``.
+        cleanup: bool = True,
```

Comment:
Why do we need to remove white spaces in a default value (i.e., implicitly use whitespaces as delimiters )?

Reply:
Thanks. It would be better to rename `cleanup` to `strip` in this case.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/136/commits/67eae0dbcfc03ac5c5d26c3c3ad8a8ca9bfbd70b.